src/timer/TimingInterface.ts

```diff
@@ -0,0 +1,148 @@
+class TimingInterface {
+    initialTime: number;
```

Comment:
Make these private variables by appending private to them.
It is also convention to append an underscore to the variable name if it is private.
i.e. private _initialTime: number